src/commands/install-modules.yml

```diff
@@ -0,0 +1,70 @@
+description: >
+  Install your node modules with automated caching and best practices applied.
```

Comment:
Can we change this command/description to "install-packages"? Node.js doesn't use the term modules for the individual packages (unlike Go which does).

This is the help text when you run `npm help install`:

> npm-install - Install a package

Reply:
Also, this question isn't specific to this PR, but does the Orb Registry Search search through these descriptions for commands? Or just the description at the orb level? cc @lokst

Reply:
That we can do 👍 thank you